lib/flipper/api/json_params.rb

```diff
@@ -0,0 +1,28 @@
+module Flipper
+  module Api
+    class JsonParams
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        request = Rack::Request.new(env)
+        if request.content_type == 'application/json'
+          body = request.body.read
+          request.body.rewind
+          update_params!(request, body)
```

Comment:
That is a great point. Will change this and keep that in mind for the future thanks!